doc/explanation/scopes.rst

```diff
@@ -0,0 +1,36 @@
+Dune projects and workspaces
+============================
+
+Whenever Dune builds anything, it does so at the level of a *Dune workspace*. A
+Dune workspace is a set of Dune projects. A typical workspace consists of a
+single Dune project.
+
+A Dune project is defined by the presence of a
+:doc:`/reference/dune-project/index` file. Each Dune project extends over the
+subtree rooted at the directory containing the
```

Comment:
"subtree" sounds like we are in a subdirectory when instead we are really talking about the directory at which the dune-project exists. I would probably reword that to "file tree".

Reply:
Amended, thanks.